### DISCUSSION TITLE
Question: React/Javascript

### DISCUSSION THREAD

**Discussion by DamezNZ** [pkgxdev/pkgx#917] [Help] [answered]
can we use react/javascript to develop on tea?

**Comment by miguelassun** [comment]
Hey damenz ,i would like to know too , thanks

**Reply by khali245** [reply]
Very Interesting

**Comment by michaelessiet** [comment]
Hey DamezNZ, please can you clarify what you mean by "develop on tea"

**Comment by sambojix** [comment]
hello DamezNZ, maybe we can only react to develop with github only

**Comment by chidubemOkafor** [comment]
I don't understand this question

**Comment by Mannnie** [comment]
The question is confusing though.

**Comment by seanparkcrypto** [comment]
lol me neither

**Comment by flaapzy** [comment]
you mean the language used to build tea?

**Comment by georgecostanza0669** [comment]
hey man, if what you mean is a 'pull request' this is possible but you must have your Discord handle in your GitHub profile. This unlocks the contributor section of the discord. Hope I helped!

**Comment by Rganthong** [comment]
What you mean brother?

**Comment by ericnelson99** [comment]
Good question, Id love to know too

**Comment by Saitama421** [comment]
say wha?

**Comment by codewithneeraj** [comment]
Hi @DamezNZ @miguelassun @khali245 @sambojix @ericnelson99 @michaelessiet 

To clear all your doubts.

The tea protocol currently supports projects listed in APT, Crates, Homebrew, npm, pkgx, PyPI and RubyGems, with more to come in the future. 

So If you have any package deployed on (APT, Crates, Homebrew, npm, pkgx, PyPI and RubyGems) your project may be qualifies to earn rewards on tea.

To check your Project eligibility follow these steps
1. GO to https://tea.xyz/rewards-for-oss-contributions
2. Type your package name in the search bar and check if  it qualifies to earn rewards on tea.

**Give upvote if it answers your question.** 
Thanks!

**Comment by Edotamara26** [comment]
I'am not yet deploy.

**Comment by Edotamara26** [comment, marked as answer]
> can we use react/javascript to develop on tea?

Yes
